nodejs-server/src/index.ts

```diff
@@ -0,0 +1,66 @@
+import * as protoLoader from '@grpc/proto-loader';
+import * as grpc from 'grpc';
+import * as path from 'path';
+import * as yargs from 'yargs';
```

Comment:
If all you're using here is the argument parser, I think this ships in it's own smaller package cc @bcoe

Reply:
:wave: hey, just use https://www.npmjs.com/package/yargs-parser in these cases; it's the parser used by yargs, without the CLI output bits.

Reply:
Thanks! Fixed.